client/js/board.js

```diff
@@ -76,22 +75,22 @@ export default class Board extends EventEmitter{
 
 	createScoreboard() {
 		this.scoreboard = new Scoreboard();
-
 		this.scoreboard.playersContainer.appendTo('#scoreboard');
 	}
 
-	render() {
-		this.intervalId = setInterval(() => {
-			this.snakes.forEach(snake => {
-				if(!snake.dead) {
-					snake.move(this);
-				}
-			});
-			this.scoreboard.updateScores(this.snakes, this.clientLocalSnake);
-			this.checkSnakeSelfCollision();
-			this.checkCollisionWithApples();
-            // END TEMP
-		}, constant.DELAY);
+	render(snakes, apples){
```

Comment:
I believe the board already has the snakes and apples as property. It shouldn't need them as arguments